app/graphql/types/mutation_type.rb

```diff
@@ -46,6 +46,7 @@ class MutationType < Types::BaseObject
     field :destroy_payment_provider, mutation: Mutations::PaymentProviders::Destroy
 
     field :add_gocardless_payment_provider, mutation: Mutations::PaymentProviders::Gocardless
+    field :add_adyen_payment_provider, mutation: Mutations::PaymentProviders::Adyen
```

Comment:
GraphQL/OrderedFields: Fields should be sorted in an alphabetical order within their section. Field `add_adyen_payment_provider` should appear before `add_gocardless_payment_provider`.